docs/corfu-queue/corfu-queue-design.md

```diff
@@ -1,12 +1,11 @@
 The class `CorfuQueue` implements a persisted queue over the abstraction of a CorfuTable. CorfuTable that only uses a HashMap<> to represent the materialized view of a distributed map does not carry a notion of ordering natively. However when implemented over the abstraction of a distributed shared log, the elements added to the map do, in fact, have an ordering imposed by their append or transaction commit operations into the global log. The class `CorfuQueue` attempts to expose this inherent ordering as a persisted Queue with three simple apis:
 
 ###  1. `CorfuRecordId enqueue(E)`
-Since a map has a key and value, where key is a conflict parameter, enqueue generates a non-conflicting Long as the key and inserts the Entry as a value into a CorfuTable. The generated Long is packed into the LSB of a UUID and returned to the caller as a CorfuRecordId. Note that this Id does not carry ordering since the operation could be part of a transaction that has not committed yet.
+Since a map has a key and value, where key is a conflict parameter, enqueue generates a non-conflicting Long as the key and inserts the Entry as a value into a CorfuTable. If enqueue() is wrapped in a Corfu transaction this CorfuRecordId returned will capture the transaction's commit address and together with the entryId define the global cluster-wide order of the entry in the Queue.
```

Comment:
I think you want to say transaction's commit order and probably not address.

Reply:
done. thanks